radicle-httpd/src/api/v1/projects.rs

```diff
@@ -416,15 +416,30 @@ async fn readme_handler(
 async fn issues_handler(
     State(ctx): State<Context>,
     Path(project): Path<Id>,
-    Query(qs): Query<PaginationQuery>,
+    Query(qs): Query<CobsQuery<issue::State>>,
 ) -> impl IntoResponse {
-    let PaginationQuery { page, per_page } = qs;
+    let CobsQuery {
+        page,
+        per_page,
+        state,
+    } = qs;
     let page = page.unwrap_or(0);
     let per_page = per_page.unwrap_or(10);
+    let state = state.unwrap_or_default();
     let storage = &ctx.profile.storage;
     let repo = storage.repository(project)?;
     let issues = issue::Issues::open(&repo)?;
-    let mut issues: Vec<_> = issues.all()?.filter_map(|r| r.ok()).collect::<Vec<_>>();
+    let mut issues: Vec<_> = issues
+        .all()?
+        .filter_map(|r| {
+            let (id, issue, clock) = r.ok()?;
+            if issue.state() == &state {
+                Some((id, issue, clock))
+            } else {
+                None
+            }
```

Comment:
This pattern is captured in a method called `then_some`, if you're interested in using it :)
```suggestion
            (issue.state() == &state).then_some((id, issue, clock))
```